projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -22,17 +22,17 @@ export class DataLayerColorService {
     }
   }
 
-  setAnnotationColor(annotation: any, color: string): void {
+  setAnnotationColor(annotation: any, color?: string): void {
     const line = annotation;
     line.backgroundColor = color + '33'; // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
   }
 
   getAnnotationColor(annotation: any): string | undefined {
     const color = annotation.backgroundColor;
-    if (typeof color === 'string') {
-      return color;
+    if (typeof color === 'string' && !color.includes('33')) {
+      return color + '33';
     }
-    return undefined;
+    return color;
   }
```

Comment:
@amolht please revert this change to color service. 
1. The getter should not be changing the value that it gets.
2. `includes()` will match a '33' anywhere in the color string.